src/Command/PullRequest/PullRequestMergeCommand.php

```diff
@@ -141,8 +144,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $gitConfigHelper->ensureRemoteExists($targetRemote, $targetRepository);
         $gitConfigHelper->ensureRemoteExists($sourceRemote, $sourceRepository);
 
+        $styleHelper->title(sprintf('Merging pull-request #%d - %s', $prNumber, $pr['title']));
+        $styleHelper->text(
+            [
+                sprintf('Source: %s/%s', $sourceRemote, $sourceBranch),
+                sprintf('Target: %s/%s', $targetRemote, $targetBranch),
+            ]
+        );
+
+        if ($squash) {
+            $styleHelper->note('This pull-request will be squashed before merging.');
+        }
+
+        $styleHelper->writeln('');
```

Comment:
This is to ensure an empty line, can be removed once https://github.com/symfony/symfony/pull/14623 is resolved.